### PR TITLE
Translate documentation to Dutch (NL)

### PR DESCRIPTION
**Pull request by anthonyringoet**
# Description

### Contribution Checklist:

- [x] **The pull request only addresses one issue or adds one feature.**
- [x] **The pull request does not introduce any breaking changes**
- [x] **I have added screenshots or gifs to help explain the change if applicable.**
- [x] **I have read the [contribution guidelines](https://github.com/usebruno/bruno/blob/main/contributing.md).**
- [x] **Create an issue and link to the pull request.**

Note: Keeping the PR small and focused helps make it easier to review and merge. If you have multiple changes you want to make, please consider submitting them as separate pull requests.

### Publishing to New Package Managers

Please see [here](../publishing.md) for more information.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -10,7 +10,7 @@
 [![Website](https://img.shields.io/badge/Website-Visit-blue)](https://www.usebruno.com)
 [![Download](https://img.shields.io/badge/Download-Latest-brightgreen)](https://www.usebruno.com/downloads)
 
-**English** | [Українська](docs/readme/readme_ua.md) | [Русский](docs/readme/readme_ru.md) | [Türkçe](docs/readme/readme_tr.md) | [Deutsch](docs/readme/readme_de.md) | [Français](docs/readme/readme_fr.md) | [Português (BR)](docs/readme/readme_pt_br.md) | [한국어](docs/readme/readme_kr.md) | [বাংলা](docs/readme/readme_bn.md) | [Español](docs/readme/readme_es.md) | [Italiano](docs/readme/readme_it.md) | [Română](docs/readme/readme_ro.md) | [Polski](docs/readme/readme_pl.md) | [简体中文](docs/readme/readme_cn.md) | [正體中文](docs/readme/readme_zhtw.md) | [العربية](docs/readme/readme_ar.md) | [日本語](docs/readme/readme_ja.md)
```
Comment (pietrygamat) [inline readme.md:13]: You may want to update the links in all translated documents as well.

Comment (anthonyringoet) [reply readme.md:13]: Added updates

File: docs/readme/readme_nl.md [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+[![Commit Activity](https://img.shields.io/github/commit-activity/m/usebruno/bruno)](https://github.com/usebruno/bruno/pulse)
+[![X](https://img.shields.io/twitter/follow/use_bruno?style=social&logo=x)](https://twitter.com/use_bruno)
+[![Website](https://img.shields.io/badge/Website-Visit-blue)](https://www.usebruno.com)
+[![Download](https://img.shields.io/badge/Download-Latest-brightgreen)](https://www.usebruno.com/downloads)
+
+[English](../../readme.md) | [Українська](docs/readme/readme_ua.md) | [Русский](docs/readme/readme_ru.md) | [Türkçe](docs/readme/readme_tr.md) | [Deutsch](docs/readme/readme_de.md) | ** Nederlands ** | [Français](docs/readme/readme_fr.md) | [Português (BR)](docs/readme/readme_pt_br.md) | [한국어](docs/readme/readme_kr.md) | [বাংলা](docs/readme/readme_bn.md) | [Español](docs/readme/readme_es.md) | [Italiano](docs/readme/readme_it.md) | [Română](docs/readme/readme_ro.md) | [Polski](docs/readme/readme_pl.md) | [简体中文](docs/readme/readme_cn.md) | [正體中文](docs/readme/readme_zhtw.md) | [العربية](docs/readme/readme_ar.md) | [日本語](docs/readme/readme_ja.md)
```
Comment (pietrygamat) [inline docs/readme/readme_nl.md:13]:
```suggestion
[English](../../readme.md) | [Українська](docs/readme/readme_ua.md) | [Русский](docs/readme/readme_ru.md) | [Türkçe](docs/readme/readme_tr.md) | [Deutsch](docs/readme/readme_de.md) | **Nederlands** | [Français](docs/readme/readme_fr.md) | [Português (BR)](docs/readme/readme_pt_br.md) | [한국어](docs/readme/readme_kr.md) | [বাংলা](docs/readme/readme_bn.md) | [Español](docs/readme/readme_es.md) | [Italiano](docs/readme/readme_it.md) | [Română](docs/readme/readme_ro.md) | [Polski](docs/readme/readme_pl.md) | [简体中文](docs/readme/readme_cn.md) | [正體中文](docs/readme/readme_zhtw.md) | [العربية](docs/readme/readme_ar.md) | [日本語](docs/readme/readme_ja.md)
```
The `**` are supposed to be markdown for bold, but as is, they are just literal asterisks
Screenshot from 2024-05-30 21-58-23

Comment (pietrygamat) [reply docs/readme/readme_nl.md:13]: Also, the links to other translations are supposed to be relative to your document, and not to root - so no `(docs/readme/readme_kr.md)` but `(./readme_kr.md)` - (they 404 now). I guess it's messed up throughout the entire repo, but make at least yours work well ;)

File: docs/readme/readme_nl.md [hunk 3]
```diff
@@ -0,0 +1,160 @@
...
+
+Je kunt Git of elke versiebeheertool naar keuze gebruiken om samen te werken aan je API-collecties.
+
+Bruno is uitsluitend offline. Er zijn geen plannen om ooit cloud-synchronisatie aan Bruno toe te voegen. We waarderen je gegevensprivacy en geloven dat deze op je apparaat moet blijven. Lees onze langetermijnvisie [hier](https://github.com/usebruno/bruno/discussions/269)
+
+[Download Bruno](https://www.usebruno.com/downloads)
+
+📢 Bekijk onze recente presentatie op de India FOSS 3.0 Conference [hier](https://www.youtube.com/watch?v=7bSMFpbcPiY)
+
+![bruno](assets/images/landing-2.png) <br /><br />
```
Comment (pietrygamat) [inline docs/readme/readme_nl.md:27]: All image links are dead - must be relative to your document

File: docs/readme/readme_nl.md [hunk 4]
```diff
@@ -0,0 +1,160 @@
...
+
+<div align="center">
+    <a href="https://github.com/usebruno/bruno/graphs/contributors">
+        <img src="https://contrib.rocks/image?repo=usebruno/bruno" />
+    </a>
+</div>
+
+### Licentie 📄
+
+[MIT](license.md)
```
Comment (pietrygamat) [inline docs/readme/readme_nl.md:160]: This is also a dead link

File: readme.md (hunk 1 above)
Comment (pietrygamat) [reply readme.md:13]: Any reason to skip `contributing`: `cn`, `de`, `ro`, `ru`, `ua`, `zhw` ? 
I understand `es`, `it`, `pt_br` - as they have no links, their problem.

File: docs/readme/readme_nl.md [hunk 5]
```diff
@@ -0,0 +1,160 @@
...
+
+Als je Bruno leuk vindt en ons open-source werk wilt ondersteunen, overweeg dan om ons te sponsoren via [GitHub Sponsors](https://github.com/sponsors/helloanoop).
+
+### Deel Getuigenissen 📣
+
+Als Bruno je heeft geholpen op je werk en in je teams, deel dan je [getuigenissen op onze GitHub-discussie](https://github.com/usebruno/bruno/discussions/343).
+
+### Publiceren naar Nieuwe Pakketbeheerders
+
+Zie [hier](docs/publishing/publishing_nl.md) voor meer informatie.
```
Comment (pietrygamat) [inline docs/readme/readme_nl.md:125]: Also a dead link

File: docs/readme/readme_nl.md [hunk 6]
```diff
@@ -0,0 +1,160 @@
...
+
+`Bruno` is een handelsmerk in bezit van [Anoop M D](https://www.helloanoop.com/).
+
+**Logo**
+
+Het logo is afkomstig van [OpenMoji](https://openmoji.org/library/emoji-1F436/). Licentie: CC [BY-SA 4.0](https://creativecommons.org/licenses/by-sa/4.0/)
+
+### Bijdragen 👩‍💻🧑‍💻
+
+Ik ben blij dat je Bruno wilt verbeteren. Bekijk de [bijdragegids](contributing.md).
```
Comment (pietrygamat) [inline docs/readme/readme_nl.md:146]: Dead link. May want to link to nl version.

**Comment by helloanoop** [comment]
Closing this PR as this has been taken care of in 6d239929dad467137a1d8ab0e40cee174304903e

**Pull request closed without merging**